src/main/java/frc/robot/subsystems/BallChucker9000.java

```diff
@@ -80,28 +91,40 @@ public void update() {
 
     // ESC
     public void rotatorMotorControl(double power) {
-        rotatorMotor.set(power);
+        if (getRotatorAtZeroSwitch()) { // Only set power if the limit switch isn't pressed
+            rotatorMotor.set(power);
+        } else {
+            rotatorMotor.set(0.0);
+        }
+    }
+
+    public void rotatorPIDControl(double setPoint) {
+        rotatorMotorControl(rotatorPID.calculate(getRotatorDistance(), setPoint));
     }
 
     public void flywheelMotorControl(double power) {
-        flywheelMotor.set(power);
+        rightFlywheelMotor.set(power);
+        leftFlywheelMotor.set(power);
     }
 
-    public void indexerMotorControl(double power) {
-        indexerMotor.set(power);
+    public void flywheelPIDControl(double velocity) {
+        flywheelMotorControl(flywheelPID.calculate(getFlywheelSpeed(), velocity));
     }
 
-    // Pistons
-    public void indexerPistonOut(boolean state) {
-        indexerPiston.set(state);
+    public void indexerMotorControl(double power) {
+        indexerMotor.set(power);
     }
 
 
     // Getters
-    public double getFlywheelEncoder() {
+    public double getFlywheelSpeed() {
         return flywheelEncoder.getRate();
     }
 
+    public double getRotatorDistance() {
```

Comment:
I would like this to return encoder.getDistance plus some constant to have it zeroed at the straight forward position. The constant could equal 120 for now.

Reply:
Also this could be renamed to getRotatorAngle() for clarity.